### PR TITLE
DFPL-265 Fix reading shutter secret

### PR DESCRIPTION
**Pull request by jasonpaige**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DFPL-265

### Change description ###
Fixes reading the shuttering config value from KV

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -130,6 +129,20 @@ def teardownSecretsForIntegrationTests() {
   env.INTEGRATION_TEST_DOCMOSIS_TORNADO_OUTPUT_FOLDER = ''
 }
 
+def setupShutteringSecret(pipelineConf) {
+  withSubscription(env.ENV != 'prod' ? 'nonprod' : 'prod') {
+    withTeamSecrets(pipelineConf, env.ENV) {
```
Comment (timja) [inline Jenkinsfile_CNP:134]: you don't need `withTeamSecrets` and `azureKeyVault`
```suggestion
```

File: Jenkinsfile_CNP [hunk 2]
```diff
@@ -130,6 +129,20 @@ def teardownSecretsForIntegrationTests() {
   env.INTEGRATION_TEST_DOCMOSIS_TORNADO_OUTPUT_FOLDER = ''
 }
 
+def setupShutteringSecret(pipelineConf) {
+  withSubscription(env.ENV != 'prod' ? 'nonprod' : 'prod') {
```
Comment (timja) [inline Jenkinsfile_CNP:133]: i don't think you'll need this either, I would try without
```suggestion
```

File: Jenkinsfile_CNP [hunk 3]
```diff
@@ -130,6 +129,16 @@ def teardownSecretsForIntegrationTests() {
   env.INTEGRATION_TEST_DOCMOSIS_TORNADO_OUTPUT_FOLDER = ''
 }
 
+def setupShutteringSecret(pipelineConf) {
```
Comment (timja) [inline Jenkinsfile_CNP:132]:
```suggestion
def setupShutteringSecret() {
```

File: Jenkinsfile_CNP [hunk 4]
```diff
@@ -259,6 +268,7 @@ withPipeline(type, product, component) {
     if (env.ENV != 'preview') {
       // For Preview the setup is performed before the smoke test since the environment could be not bootstrapped yet (if the first build or removed the PR helm release)
       echo "CCD definition upload for environment: ${env.ENV}"
+      setupShutteringSecret(pipelineConf)
```
Comment (timja) [inline Jenkinsfile_CNP:271]:
```suggestion
      setupShutteringSecret()
```

**Review by timja — COMMENTED** [review]
unused variable, you could also do a separate PR to clean up all the other functions probably

**Review by timja — APPROVED** [review]

**Review by lewisbirks — APPROVED** [review]

**Pull request merged**